prospector_profile.yaml

```diff
@@ -1,15 +1,5 @@
 ignore-paths:
-  - api/v1/views/post_boot.py
-  - core/models/project.py
-  - core/models/t.py
-  - scripts/admin_clean_provider_machine.py
-  - service/accounts/aws.py
-  - service/accounts/eucalyptus.py
-  - service/instance.py
-  - api/v1/views/identity_membership.py
-  - api/v1/views/meta.py
   - settings/local.py
-  - settings/secrets.py
```

Comment:
Can we please also exclude `secrets.py`? Really anything under `atmosphere/settings`.

Reply:
That's removed in a later commit

Reply:
I have a plan to remove `settings/local.py`, also, but will require more work. Essentially I want one file of variables `settings/__init__.py` templated by clank.

Reply:
Ahah! OK. I'll stand by.